### DISCUSSION TITLE
Request Limitation does it support over  500 000 request per day?

### DISCUSSION THREAD

**Discussion by umrzoq-toshkentov** [osm-search/Nominatim#3070] [Q&A] [answered]
How many limit of requests are available here? Does it support over 500 000 request per day from one host 

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mtmail** [comment, marked as answer]
nominatim.openstreetmap.org runs on donated hardware with limited capacity. 500000 is too much. See https://operations.osmfoundation.org/policies/nominatim/

You can install your own server with Nominatim software https://nominatim.org/release-docs/latest/admin/Installation/ That's about 10 requests per second, a medium size server can handle that. All depends on which data (which regions of the world) you import.

Or use a third-party company https://wiki.openstreetmap.org/wiki/Nominatim#Alternatives_.2F_Third-party_providers They usually charge money for their service.

You asked the same question in the leaflet-geosearch project. Note that Nominatim software does not support autocomplete/type-ahead, so a search for "New Yor" will not return "New York".
